src/store/modules/invites.js

```diff
@@ -0,0 +1,67 @@
+import Vue from 'vue';
+import Ae from '@aeternity/aepp-sdk/es/ae/universal';
+import Node from '@aeternity/aepp-sdk/es/node';
+import MemoryAccount from '@aeternity/aepp-sdk/es/account/memory';
```

Comment:
I think we should prefer importing from the index file:
```suggestion
import { Universal as Ae, Node, MemoryAccount } from '@aeternity/aepp-sdk/es';
```
this will make it easier to refactor files in sdk 🙂